api/tsconfig.eslint.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": [
+    "node_modules",
+    "*.js",
+    "webpack-node.config.js",
+    "yarn.lock"
+  ],
+  "include": [
```

Comment:
Do we have to include every single directory explicitly? Can we not do a `**/*` type pattern? This will be hard to maintain otherwise...

Reply:
Ahh we because only want to lint against the specified folders inside the `/api` folder